data/events.xml

```diff
@@ -1,5 +1,136 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <events>
+    <event>
+        <lang>en</lang>
+        <startDate>2022-02-06</startDate>
+        <endDate>2022-02-06</endDate>
+        <location>Brussels, Belgium</location>
```

Comment:
@TinaNEs Fosdem is fully online this year. Please remove the location and add the "online" tag